components/heatmap/HeatMap.vue

```diff
@@ -356,25 +360,34 @@ export default {
     },
 
     apply_heatmap_settings: function () {
-      this.$store.commit("heatmap/set_auto_scale", this.autoscale);
-
       if (this.is_apply_set) {
```

Comment:
Any thoughts on if we would want to prevent a user from being able to apply any settings until live view is active? Versus allow users to apply settings for when they start live view?

Reply:
@tannermpeterson Hamed wants the apply button and for no changes to be made until then.

Reply:
I think it would make sense to disable the apply and reset buttons while live view isn't running. The alternative would be to try and make it so that the values don't get all funky (NaN, undefined, etc.) if settings are applied while the live view isn't running and the heatmap doesn't have any values stored

Reply:
@tannermpeterson  Yes, I did get that fixed with no NaN values.. there were some definite bugs that were causing that. So that should be good to go. But yes, I can add that additional condition in for disabling the button